docs/topics/auth/customizing.txt

```diff
@@ -898,11 +898,13 @@ the User model that is being used by a project. Any test that instantiates an
 instance of User will fail if the User model has been swapped out. This
 includes any attempt to create an instance of User with a fixture.
 
-To ensure that your test suite will pass in any project configuration,
-``django.contrib.auth.tests.utils`` defines a ``@skipIfCustomUser`` decorator.
-This decorator will cause a test case to be skipped if any User model other
-than the default Django user is in use. This decorator can be applied to a
-single test, or to an entire test class.
+.. deprecated:: 1.9
```

Comment:
I'd leave the existing paragraph and add `.. deprecated:: 1.9` after it with an explanation of why it's no longer necessary.